test/scripts/amalgamate-sdsl.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Reply:
This will force a lookup in the PATH variable for bash, for "interesting" system setups